docs/content/users/extend/custom-tls-certificates.md

```diff
@@ -3,5 +3,9 @@
 It’s possible to use “real” TLS certificates issued by a CA rather than the local-development-oriented `mkcert` command.
 
 1. Obtain a certificate and key from Let’s Encrypt or another source.
-2. Install the certificate and key in your project’s `.ddev/custom_certs` directory. Each certificate must be named with the pattern `fqdn.crt` and `fqdn.key`. If you’re working with a project `example.ddev.site`, you would have `example.ddev.site.crt` and `example.ddev.site.key` in `.ddev/custom_certs`. There must be one cert-set for each FQDN handled by the project.
+2. Install the certificate and key in your project’s `.ddev/custom_certs` directory.
+   * If using traditional router (current default), each certificate must be named with the pattern `fqdn.crt` and `fqdn.key`. If you’re working with a project `example.ddev.site`, you would have `example.ddev.site.crt` and `example.ddev.site.key` in `.ddev/custom_certs`. There must be one cert-set for each FQDN handled by the project.
+   * If using Traefik (`ddev config global --use-traefik`) then the files should be named `<projectname>.crt` and `<projectname>.key`, for example `exampleproj.crt` and `exampleproj.key`.
```

Comment:
Replace redundant instruction with link:

```suggestion
   * If you’re [using Traefik](../configuration/experimental.md#traefik-router), the files should be named `<projectname>.crt` and `<projectname>.key`, for example `exampleproj.crt` and `exampleproj.key`.
```

Because there’s more to know about Traefik in general, and in this context the focus is on custom certificates.